extensions/notebook/src/telemetry.ts

```diff
@@ -0,0 +1,24 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import AdsTelemetryReporter from 'ads-extension-telemetry';
+
+const packageJson = require('../package.json');
+export const TelemetryReporter = new AdsTelemetryReporter(packageJson.name, packageJson.version, packageJson.aiKey);
+
+export const NotebookTelemetryView = 'Notebook';
```

Comment:
The view is supposed to be the location where the action occurs. So like which dialog, wizard, etc. Since all the events you're adding are for the tree view I'd suggest naming this something like BookView or something similar